pkgs/top-level/all-packages.nix

```diff
@@ -27831,6 +27831,8 @@ in
 
   btcpayserver = callPackage ../applications/blockchains/btcpayserver { };
 
+  charge-lnd = callPackage ../applications/blockchains/charge-lnd.nix { };
```

Comment:
```suggestion
  charge-lnd = callPackage ../applications/blockchains/charge-lnd { };
```

Reply:
That only works if charge-lnd were a directory with default.nix in it - is that the suggestion?

Reply:
Well not every one but it seems to be the prevailing style, fair enough. Fixed in bce3e72.